backend/typescript/package.json

```diff
@@ -18,6 +18,7 @@
     "express": "^4.17.1",
     "firebase-admin": "^9.5.0",
     "mongoose": "^5.11.17",
+    "node-fetch": "^2.6.1",
```

Comment:
`note-fetch` or `axios`?

Reply:
I think both are ok, so we can keep it as is